code/modules/food_and_drinks/recipes/processor_recipes.dm

```diff
@@ -73,22 +73,26 @@
 	multiplier = 3
 	blacklist = null
 
-/datum/food_processor_process/cutlet/chicken
-	input = /obj/item/food/meat/cutlet/chicken
-	output = /obj/item/food/raw_meatball/chicken
```

Comment:
Is this removal of the food processor "recipe" for chick meatballs intentional?

Reply:
Uhhh.... Yes, but because you seem to like it, I will bring it back in. Please believe me.

Reply:
In all honesty I thought I copy-pasted chicken meatball recipe during to just reading the output for both recipes.